wasm-and-containers/workshop/03-deploy-spin-to-k8s.md

```diff
@@ -30,16 +30,25 @@ In addition to the requirements from the previous sections, you'll need the foll
 - [k3d](https://k3d.io/v5.4.6/#installation)
 - [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
 
+Install this Spin plugin for Kubernetes:
+
+- [Spin k8s plugin](https://github.com/chrismatteson/spin-plugin-k8s)
+
+```
+$ spin plugin install -y -u https://raw.githubusercontent.com/chrismatteson/spin-plugin-k8s/main/k8s.json
+```
 
 ## 1. Create and Configure the k3d cluster
 
+> See [The Spin Kubernetes docs](https://developer.fermyon.com/spin/v2/kubernetes) for more information on configuring various Kubernetes flavors.
+
 The first step is to create a k3d cluster with the wasm shims installed:
 
 ```bash
-k3d cluster create wasm-cluster --image ghcr.io/deislabs/containerd-wasm-shims/examples/k3d:v0.9.2 -p "8081:80@loadbalancer" --agents 2
+k3d cluster create wasm-cluster --image ghcr.io/deislabs/containerd-wasm-shims/examples/k3d:v0.10.0 -p "8081:80@loadbalancer" --agents 2 --registry-create mycluster-registry:12345
```

Comment:
Should we bump to the latest image? https://github.com/deislabs/containerd-wasm-shims/blob/main/containerd-shim-spin/quickstart.md#start-and-configure-a-k3d-cluster
```suggestion
k3d cluster create wasm-cluster --image ghcr.io/deislabs/containerd-wasm-shims/examples/k3d:v0.11.1 -p "8081:80@loadbalancer" --agents 2 --registry-create mycluster-registry:12345
```